src/utils/setup.ts

```diff
@@ -70,32 +70,36 @@ export function initInstallation(): Promise<Parse.Installation> {
   }
   const promise: Promise<Parse.Installation> = new Promise(
     (resolve, reject) => {
-      PushNotifications.addListener("registration", (token: Token) => {
-        generateInstallation({ deviceToken: token.value })
-          .then((x: Parse.Installation) => resolve(x))
-          .catch((x: any) => reject(x));
-      });
+      if (Capacitor.isPluginAvailable("PushNotifications")) {
```

Comment:
Description should probably mentioned this fix being in this PR,  too.

Reply:
separate PR https://github.com/TeamFranka/affinity/pull/190

Reply:
fixed by https://github.com/TeamFranka/affinity/pull/190